### PR TITLE
Update Algorithm Pig Latin

### PR DESCRIPTION
**Pull request by dakshshah96**
Refer #1094

### CODE REVIEW

File: Algorithm-Pig-Latin.md [hunk 1]
```diff
@@ -117,40 +157,33 @@ translatePigLatin("consonant");
 
 :rocket: [Run Code](https://repl.it/CLmv/0)
 
-### Fourth Solution (recursive and declarative)
```
Comment (Rafase282) [inline Algorithm-Pig-Latin.md]: I would prefer this answer over any of the others, the other 3 uses regex, this is the only one that does not.

Comment (dakshshah96) [inline Algorithm-Pig-Latin.md]: True! Alright, I'll make the necessary changes. Should I put this as the Basic solution?

**Comment by Rafase282** [comment]
I would say drop the second solution from the original so they would go in this order, 1,3,4 as beginer, intermediate, advance.

**Comment by dakshshah96** [comment]
@Rafase282 I've implemented the suggestions. Hope it's alright!

**Comment by Rafase282** [comment]
Great, thanks!

**Pull request merged**